build.sbt

```diff
@@ -68,7 +68,7 @@ lazy val `akka-discovery-aws-api-async` = project
   .settings(
     name := "akka-discovery-aws-api-async",
     organization := "com.lightbend.akka.discovery",
-    Dependencies.DiscoveryAwsApiAsync
+    libraryDependencies := Dependencies.DiscoveryAwsApiAsync
```

Comment:
I prefer `++=` as a general style (since it makes builds more composable), though in this case it probably doesn't really matter.

Reply:
I like the WYSIWYG of `:=`. The old setup injecting libraries from `Common` was a bit surprising to me.